### ISSUE TITLE
[BUG] Mention changelog/release notes location?

### ISSUE THREAD

**Issue by phlummox** [Jermolene/TiddlyWiki5#4810]
**Description**

Not a bug with the code, but perhaps with the documentation. I couldn't see a [changelog](https://keepachangelog.com/en/1.0.0/) for the project, and it took me a while to work out where the Release notes are (i.e., under https://github.com/Jermolene/TiddlyWiki5/tree/master/editions/tw5.com/tiddlers/releasenotes).

This might be obvious to people familiar with the codebase, but for newcomers, perhaps it might be worth mentioning this in the [README][curr-readme]? (Or perhaps include a traditional `Changelog` file - but containing a pointer to that directory - since that's where people most likely will look.)

[curr-readme]: https://github.com/Jermolene/TiddlyWiki5/blob/8cf458d3b3f1e38c14a2819529e08dca4a7e297c/readme.md

**Comment by pmario** [comment]
It's in the right sidebar TOC : About : Releases + Alpha + Beta. see: https://tiddlywiki.com/#Releases:Releases%20BetaReleases%20AlphaReleases

Those release notes are manually created and give a very good overview, how TW evolved. Every release note also contains a link to github compare, where you can see the different commits.

**Comment by pmario** [comment]
But you are right. the README could do with a little bit of love.

**Comment by phlummox** [comment]
Indeed, but if someone comes across the GitHub repository first, they won't necessarily have seen the site or sidebar  :) And it's pretty standard to at least mention in the codebase where your release notes or changelog are to be found - even with just a link.

Anyway - happy to do a pull request to amend the README by adding a link to the release notes, if you'd like.

**Comment by pmario** [comment]
> Anyway - happy to do a pull request to amend the README by adding a link to the release notes, if you'd like.

Be aware, that the README is auto-generated by the build process. So the tiddler to modify is: https://github.com/Jermolene/TiddlyWiki5/blob/master/editions/tw5.com/tiddlers/readme/ReadMe.tid
